podcasts/New Search/Views/History/SearchHistoryCell.swift

```diff
@@ -0,0 +1,76 @@
+import SwiftUI
+import PocketCastsDataModel
+import PocketCastsUtils
+
+struct SearchHistoryCell: View {
+    @EnvironmentObject var theme: Theme
+
+    var podcast: Podcast?
+
+    var episode: Episode?
+
+    var searchTerm: String?
```

Comment:
As mentioned previously, `Episode` and `Podcast` won't be used for displaying the results/history. It will be another entity. When we have this in place then we can make this change. Wdyt?